int32 lidar_process_start
---
float32[] lidar_results